{
  underTest.setResult(mock(CeTaskResult.class));
  expectedException.expect(IllegalStateException.class);
  expectedException.expectMessage("CeTaskResult has already been set in the holder");
  underTest.setResult(mock(CeTaskResult.class));
}
